src/Universe.cpp

```diff
@@ -53,7 +53,7 @@ Universe::Universe(const int id, const char* name) {
 
   /* If the user did not define an optional ID, create one */
   if (id == -1)
-    _id = cell_id();
+    _id = universe_id();
```

Comment:
I also see the default for cell id in the constructor is 0 while the default universe id in the constructor is -1. I feel like they should both be -1 (or at least the same value). Agreed?